test/app-tap/module_api.c

```diff
@@ -284,6 +284,30 @@ test_fiber(lua_State *L)
 	return 1;
 }
 
+int
+fiber_set_ctx_test_func(va_list va)
```

Comment:
I suggest to make the function `static` - it is never used outside of this file.

Reply:
done